picard/util/webbrowser2.py

```diff
@@ -35,13 +35,14 @@
 
 from picard.const import PICARD_URLS
 from picard.i18n import gettext as _
+from picard.util import get_url
 
 
 def open(url):
     if url in PICARD_URLS:
         url = PICARD_URLS[url]
```

Comment:
Do we still need those lines above since we have https://github.com/metabrainz/picard/pull/2758/changes#diff-87e59aba7020204f1dc6f009745d85ead5c8114495a302fe44e435afc4293c11R785 ?

Reply:
No, sir, we do not still need those lines.  Good catch!